src/node/transaction.cpp

```diff
@@ -39,6 +39,7 @@ TransactionError BroadcastTransaction(NodeContext& node, const CTransactionRef t
 
     { // cs_main scope
     LOCK(cs_main);
+    assert(node.chainman);
```

Comment:
would be nice to assert before cs_main, because cs_main "belongs" to the chainman